README.md

```diff
@@ -19,6 +19,7 @@ A software that monitors and relayers:
 - CometBFT blocks to update Native -> Pera light client
 - Bitcoin blocks to update Bitcoin -> Native light client
 - Bitcoin SPV proofs to verify dWallet holdings
+- Native -> Bitcoin transaction relayer
```

Comment:
Please add the steps in README on how to run the script.

1. Copy `.env.example` file to `.env`
2. Update values in `.env` file
3. Run script with path of transaction file as command line argument

Reply:
Done